package.json

```diff
@@ -9,7 +9,8 @@
   },
   "scripts": {
     "postinstall": "patch-package",
-    "start": "yarn build-styling && multiview [ yarn workspace @dso-toolkit/core start --prod ] [ yarn workspace dso-toolkit start ]",
+    "start": "yarn build-styling && multiview [ yarn workspace @dso-toolkit/core start ] [ yarn workspace dso-toolkit start ]",
+    "storybook": "multiview [ yarn workspace @dso-toolkit/core start --dev ] [ yarn workspace @dso-toolkit/core storybook ]",
```

Comment:
Hier geen `build-styling` script, dat bouwt op dit moment alleen maar de SVG spritesheet benodigd voor een omgeving met CSS global styling, de oude `index.html` van Stencil heeft dat niet en is voor nu out of scope: De Storybook implementatie is slechts een vervanging van Stencil's `index.html`

In de toekomst zal er ook global styling in Storybook komen, het builden van de SVG spritesheet is dan ook nodig.